nab/labeler.py

```diff
@@ -349,6 +357,20 @@ def checkWindows(self):
             windows[i] = [windows[i][0], windows[i+1][1]]
             del windows[i+1]
 
+      length = len(self.corpus.dataFiles[relativePath].data)
+      probationIndex = int(math.ceil(self.probationaryPeriod*length))
+      probationTimestamp = self.corpus.dataFiles \
+                             [relativePath].data["timestamp"][probationIndex]
+      if num_windows > 0:
+        if (pandas.to_datetime(windows[0][0])
+            -probationTimestamp).total_seconds() < 0:
+#          raise ValueError("The first window in %s overlaps with the"
```

Comment:
No, I mean an exception. In general I don't think we should proceed without manual intervention in the case where the probationary period overlaps the first anomaly window.